message_test.go

```diff
@@ -256,6 +257,9 @@ func TestAttachmentOnly(t *testing.T) {
 	}
 
 	testMessage(t, m, 0, want)
+	if err := teardownFile("/tmp/test.pdf"); err != nil {
+		panic(err)
```

Comment:
It's generally not a good practice to panic unless absolutely necessary. Have a look at using `t.Errorf("teardown: %s", err)` instead.